tests/packagedcode/data/citation/CITATION.cff

```diff
@@ -0,0 +1,89 @@
+# This CITATION.cff file was created by ruby-cff (v 1.1.0).
+# Added few changes to help with testing.
+# Gem: https://rubygems.org/gems/cff
+# CFF: https://citation-file-format.github.io/
+# Link: https://github.com/citation-file-format/ruby-cff/blob/main/CITATION.cff
+
+cff-version: 1.2.0
+message: If you use ruby-cff in your work, please cite it using the following metadata
+title: Ruby CFF Library
+type: software
+abstract: This library provides a Ruby interface to manipulate Citation File Format files
+authors:
+- family-names: Haines
+  given-names: Robert
+  orcid: https://orcid.org/0000-0002-9538-7919
+  affiliation: The University of Manchester, UK
+- name: The Ruby Citation File Format Developers
+  email: abc@developer.com
+keywords:
+- ruby
+- credit
+- software citation
+- research software
+- software sustainability
+- metadata
+- citation file format
+- CFF
+version: 1.1.0
+doi: 10.5281/zenodo.1184077
+date-released: 2023-04-10
+license: Apache-2.0
+url: https://rubygems.org/
+repository: https://github.com/citation-file-format/
+repository-artifact: https://rubygems.org/gems/cff
+repository-code: https://github.com/citation-file-format/ruby-cff
+references:

```

Comment:
Here we have to think if references can be listed as dependencies. See https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#referencing-other-work which does support this. Maybe only for the references with a `type: software`?